src/node_file.cc

```diff
@@ -895,6 +1047,34 @@ static void Open(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+static void OpenFD(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  CHECK_GE(args.Length(), 3);
+  CHECK(args[1]->IsInt32());
+  CHECK(args[2]->IsInt32());
+
+  BufferValue path(env->isolate(), args[0]);
+  CHECK_NE(*path, nullptr);
+
+  int flags = args[1]->Int32Value();
+  int mode = static_cast<int>(args[2]->Int32Value());
```

Comment:
Can you use the non-deprecated overloads of `Int32Value()`, or just `.As<Int32>()->Value()`?

Reply:
Yep. Will make that change in Open() also